authorizer-app/docker/entrypoint.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+ROOT_DIR=/usr/share/nginx/html
+# Replace env vars in JavaScript files
+echo "Replacing env vars in JS"
+# So far, the main.*.js is the only file where the app is compiled into
+for file in $ROOT_DIR/main.*.js* $ROOT_DIR/index.html
+do
+  echo "Processing $file ...";
+
+#  sed -i 's|/\?BASE_HREF|'${BASE_HREF}'|g' $ROOT_DIR/index.html
```

Comment:
Is this no longer applied? Perhaps checking for `"s|'BASE_HREF'|'${BASE_HREF}'|g"`? Similar for the entries below.